ext/curl/config.m4

```diff
@@ -6,75 +6,97 @@ PHP_ARG_WITH(curl, for cURL support,
 [  --with-curl[=DIR]         Include cURL support])
 
 if test "$PHP_CURL" != "no"; then
-  if test -r $PHP_CURL/include/curl/easy.h; then
-    CURL_DIR=$PHP_CURL
```

Comment:
should we use pkg-config even if the path is specified explicitly? I would expect the files in `<somepath>/include/curl/easy.h` to be used regardless of whether pkg-config is available, when I compile PHP with `--with-curl=<somepath>`